playbooks/hc-ansible-deployment/tasks/sample_replace_gluster.yml

```diff
@@ -0,0 +1,7 @@
+---
+- remote_user: root
+  gather_facts: no
+  hosts: cluster_node
+  no_log: True
```

Comment:
Why no_log: true, does it log any sensitive data?

Reply:
it shows peer id, hostname and gluster vol info details